README.md

```diff
@@ -201,6 +201,19 @@ $ docker run -it -v $(pwd)/data/blockstack-api:/root/.blockstack -v /tmp/:/tmp/
 $ docker run -d -v $(pwd)/data/blockstack-api:/root/.blockstack -v /tmp/:/tmp/ -p 6270:6270 myrepo/blockstack:latest blockstack api start-foreground --password PASSWORD --debug
 ```
 
+### Running Browser in Docker
```

Comment:
So the approach you have taken is more what you would want to do with a VM. Install everything in it and allow users to enable each separate part.